### ISSUE TITLE
Publish new version to npm that uses mongodb 1.4.x driver

### ISSUE THREAD

**Issue by saintedlama** [scttnlsn/mubsub#36]
Current version on npm still uses mongodb 1.3.x driver which is not compatible with mongodb server version 3.0 while the github version was already upgraded to mongodb driver 1.4.x which is compatible to mongodb server version 3.0

**Comment by DrBeza** [comment]
I encountered the same problem with the npm version as I'm running Mongo 3.0+. Any chance the npm version could be updated?

**Comment by scttnlsn** [comment]
Are you guys successfully running the `master` version?  I'm getting a bunch of failing tests and I haven't had time to look into them.

**Comment by saintedlama** [comment]
It's running in production but the tests are failing. I'll try to get the tests working with 1.4.x

**Comment by DrBeza** [comment]
I also took some time to look at this yesterday. I ended up getting the tests passing with MongoDB ~2.0 but with considerable code changes in channel.js (mostly due to switching to use stream() method).

Unfortuantely the project I am using mubsub is, is still very much in POC stage so I was trying to prove everything is still working. I'm planning to commit my changes into my github account when I get time.
